thermalnetwork/network.py

```diff
@@ -204,11 +206,7 @@ def convert_features(self, json_data):
                 #     max_pt = upper_right_point(ghe_polygons[0])
                 #     geometric_constraints["length"] = max_pt[0] - min_pt[0]
                 #     geometric_constraints["width"] = max_pt[1] - min_pt[1]
```

Comment:
Could probably delete a bit more of this, just for cleanup sake.

Reply:
Good catch. I missed that one there.